TestFoundation/TestFileManager.swift

```diff
@@ -367,15 +367,18 @@ class TestFileManager : XCTestCase {
             let systemSize = attrs[.systemSize] as? NSNumber
             XCTAssertNotNil(systemSize)
             XCTAssertGreaterThan(systemSize!.uint64Value, systemFreeSize!.uint64Value)
-            
+
+#if !os(Windows)
```

Comment:
Instead of commenting these out, can you introduce a new variant of `shouldAttemptXFailTests(…)`, e.g. `shouldAttemptWindowsXFailTests(…)`?

Reply:
As you wish